pkg/collector/tcp/tcp.go

```diff
@@ -144,7 +193,72 @@ func (c *Collector) Build(_ *slog.Logger, _ *wmi.Client) error {
 		[]string{"af"},
 		nil,
 	)
-
+	c.connectionsCLOSED = prometheus.NewDesc(
```

Comment:
Does it make sense to have an dedicated metric for each state?

What about having one metric (connection_state) and a label "state"

Reply:
Great idea

Reply:
state label added